pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -3933,6 +3942,28 @@ public Map<String, List<InstanceInfo>> getTableToLiveBrokersMapping() {
 
     Map<String, List<InstanceInfo>> result = new HashMap<>();
     ZNRecord znRecord = ev.getRecord();
+
+    if (nullableTableName != null && !nullableTableName.toString().isEmpty()) {
+      List<String> tableNameWithType = getExistingTableNamesWithType(nullableTableName.toString(), null);
+      if (tableNameWithType.isEmpty()) {
+        throw new TableNotFoundException(String.format("Table=%s not found", nullableTableName));
```

Comment:
should we make it consistent? 
```suggestion
        throw new ControllerApplicationException(LOGGER, String.format("Table=%s not found", nullableTableName), Response.Status.NOT_FOUND);
```